packages/shared/src/components/cards/PostMetadata.tsx

```diff
@@ -25,12 +27,10 @@ export default function PostMetadata({
 
   return (
     <div
-      className={classNames(
-        'flex items-center text-theme-label-tertiary',
-        typoClassName,
-        className,
-      )}
+      className={classNames('flex items-center  ', typoClassName, className)}
     >
+      {!!username && <span>@{username}</span>}
+      {!!createdAt && !!username && <Separator />}
```

Comment:
Where was this before? 
Is it new spec on the existing one?

Reply:
I needed username here, instead of using a separate component, just changed it here.

<img width="317" alt="SCR-20230124-lv4" src="https://user-images.githubusercontent.com/1488164/214294859-1a26e301-b871-466c-a51c-e6325a2ea239.png">

Reply:
Ah cool, makes sense then 🥳